iac-extensions/terraform/src/test/resources/checks/CryptoKeyRotationPeriod/gcp.tf

```diff
@@ -0,0 +1,15 @@
+resource "google_kms_crypto_key" "k1" {
+  name            = "k1"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "100000s"
+}
+
+resource "google_kms_crypto_key" "k2" { # Noncompliant
```

Comment:
Maybe add also a message verification. This way we can prevent the message from being changed by mistake.